### DISCUSSION TITLE
[Bug]: Cannot add custom endpoint - groq

### DISCUSSION THREAD

**Discussion by kedar-1** [danny-avila/LibreChat#2614] [Troubleshooting] [answered]
### What happened?

After enable custom endpoint I'm not able add Groq.
API Key is in place.

I have replaced Anthropic and this endpoint is gone.
There is no option to choose a Groq.

### Steps to Reproduce

Uncomment ENDPOINTS
Replace Anthropic with "groq"
Created librechat.yaml : cp librechat.example.yaml librechat.yaml
Redeploy: compose down & up

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
Tried URL /api/logs but there is an error message:

Unexpected Application Error!
404 Not Found
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment]
In the .env, you don't have to uncomment `ENDPOINTS` and add groq. If you want to manually decide of its position in the list, you have to use `custom` since all cuttom endpoints provided with the librechat.yaml file falls under the same "custom" endpoint.

Also, you can look at the console logs when you restart librechat. You should have a message about the librechat.yaml file properly loading (or not)

**Comment by kedar-1** [comment]
Sorry, still dont undestard where to put or configure groq...

Anyway, I found an error log record in ./logs directory on host filesystem. 
Its not clear to me why docs refers only to /api/logs endpoint and not directory?

```
{"code":"ENOENT","errno":-2,"level":"error","message":"Config file YAML format is invalid: ENOENT: no such file or directory, open '/app/librechat.yaml'","path":"/app/librechat.yaml","stack":"Error: ENOENT: no s
uch file or directory, open '/app/librechat.yaml'\n    at Object.openSync (node:fs:596:3)\n    at Object.readFileSync (node:fs:464:35)\n    at loadYaml (/app/api/utils/loadYaml.js:6:27)\n    at loadCustomConfig
(/app/api/server/services/Config/loadCustomConfig.js:36:20)\n    at AppService (/app/api/server/services/AppService.js:20:25)\n    at startServer (/app/api/server/index.js:36:9)\n    at process.processTicksAndRe
jections (node:internal/process/task_queues:95:5)","syscall":"open"}
```

**Reply by fuegovic** [reply]
try validating your file with: https://yamlchecker.com/

**Reply by kedar-1** [reply]
OK, thanks for hint.
librechat.yaml file was and is valid.

**Reply by danny-avila** [reply, marked as answer]
@kedar-1 file may be valid, but it's not getting picked up by docker. To be sure, did you follow this step?

https://docs.librechat.ai/install/configuration/custom_config.html#docker-setup

If you have, and are using the default compose file, try:

```bash
docker compose down
docker compose up
```

**Reply by kedar-1** [reply]
OK, thanks, I was able to add a Groq model to the list.

In your link the section of librechat.yml config is different than the section in example file:
```
  api:
    volumes:
    - type: bind
      source: ./librechat.yaml
      target: /app/librechat.yaml

```

The documentaion is not clear about librechat.yml, I was assuming the librechat.yml will be included be default as was in docs:
_Note : If you prefer LibreChat to search for the configuration file in the root directory (which is the default behavior), simply leave this option commented out._

Still dont uderstand why my list of endpoints has not only list I have provided, there is a Mistral model which I don't want...
